inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -519,8 +520,22 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
             ngraphNode->constructor_validate_and_infer_types();
     }
 
+    // Check that operation is ExperimentalDetectron*
+    std::string type = params.type;
```

Comment:
I propose to avoid a code duplication. You can always change type from experimental to opset6 and use the code above to create an operation.

Reply:
According this file, `params.type` is not opset version, but is operation type (`Concat`, `Interpolate`, etc). Next, according to this file, opset version is `params.version`. But we cannot change `params.version`, because `params` is passed by a constant reference (see line 398).

Reply:
But we can create copy of `params` and change `version` attribute of this copy.